### PR TITLE
geopandas 0.13.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 
 build:
   number: 0
   # rtree is missing on s390x.
   # geopandas has unsatisfable dependencies for python 3.10 (Seems like the incompatible dependencies are
   # libgdal-3.0.2-h0f233a1_0 requires proj >=6.2.1,<6.2.2.0a0, but we have pyproj with a dependent proj >=8.2.1). 
-  # Generaly, it's not clear if geopandas 0.12.2 has support for python 3.10 on all platfroms,
-  # see https://github.com/geopandas/geopandas/blob/v0.12.2/.github/workflows/tests.yaml
-  skip: True  # [py<38 or s390x or py>=310]
+  skip: True  # [py<38 or s390x or py>310]
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml:17]:
```suggestion
  skip: True  # [py<38 or s390x]
```

**Comment by jorisvandenbossche** [comment]
Are there plans to merge this? It would be nice to make geopandas available on more recent python versions

**Comment by skupr-anaconda** [comment]
I'll close this PR as there is an incompatibility of `libgdal-3.0.2-h0f233a1_0` that requires `proj >=6.2.1,<6.2.2.0a0`, but we have `pyproj` with a dependent `proj >=8.2.1`. We should first rebuild/update `libgdal` to make it compatible with `pyproj`/`proj`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -31,10 +29,10 @@ outputs:
...
         - python
         - packaging
-        - pandas >=1.0.0
-        - pyproj >=2.6.1.post1
-        - shapely >=1.7
-        - fiona >=1.8
+        - pandas >=1.1.0
+        - pyproj >=3.0.1
+        - shapely >=1.7.1
+        - fiona >=1.8.19
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml:35]: In the conda-forge recipe, we moved fiona from `geopandas-base` to `geopandas`

**Comment by jorisvandenbossche** [comment]
> We should first rebuild/update libgdal to make it compatible with pyproj/proj

Are there plans to do so?

> I'll close this PR as there is an incompatibility of ..

In theory you could still make geopandas available, because fiona/gdal is only an optional dependency. But I understand that that is annoying because I assume the recipe by default tests that it can be installed in the env, so you need to be able to install all dependencies, also for the main `geopandas` package

File: recipe/meta.yaml (hunk 2 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:35]: Hm, but then I suspect that `pip check` will complain about missing required dependency https://github.com/geopandas/geopandas/blob/d5add48e966bb4e711d2399721df88349a358904/pyproject.toml#L30

Comment (skupr-anaconda) [reply recipe/meta.yaml:35]: I found that `import fiona` takes place only here in the try/except block https://github.com/geopandas/geopandas/blob/32825981fdd5518ea41352507d0982401a870317/geopandas/tools/_show_versions.py#L61 so moving it from `geopandas-base` to `geopandas` makes sense.

**Comment by skupr-anaconda** [comment]
> > We should first rebuild/update libgdal to make it compatible with pyproj/proj
> 
> Are there plans to do so?

I think we should investigate and fix it. I am unsure about the duration it will take.

File: recipe/meta.yaml (hunk 2 above)
Comment (jorisvandenbossche) [reply recipe/meta.yaml:35]: > Hm, but then I suspect that pip check will complain about missing required dependency

That's indeed the case, so in conda-forge we don't have a `pip check`

**Pull request closed without merging**